usr/sbin/registercloudguest

```diff
@@ -454,7 +527,6 @@ argparse.add_argument(
 
 
 def main(args):
-    global registration_returncode
```

Comment:
you dropped the scope of registration_returncode to be global. Can we be sure this is not causing an issue elsewhere in the code ? In general I like this change but I can't foresee its after effects

Reply:
I tracked the variable and I would like to remove the global one but for now, I kept it

the idea behind this refactor exercise is to isolate scopes and make them independent,
I do not want to introduce a regression, so let's keep it for this release and I will work in removing
it for the next

it is enough refactoring for one release

Reply:
sounds good, thanks